README.md

```diff
@@ -127,64 +222,63 @@ final class CustomProvider: FeatureProvider {
         // resolve a boolean flag value
     }
 
-    func getStringEvaluation(
-        key: String,
-        defaultValue: String,
-        context: EvaluationContext?
-    ) throws -> ProviderEvaluation<String> {
-        // resolve a string flag value
+    ...
+}
+
+```
+> Built a new provider? [Let us know](https://github.com/open-feature/openfeature.dev/issues/new?assignees=&labels=provider&projects=&template=document-provider.yaml&title=%5BProvider%5D%3A+) so we can add it to the docs!
+
+### Develop a hook
+
+To develop a hook, you need to create a new project and include the OpenFeature SDK as a dependency.
+This can be a new repository or included in [the existing contrib repository](https://github.com/open-feature/swift-sdk-contrib) available under the OpenFeature organization.
+Implement your own hook by conforming to the `Hook interface`.
+To satisfy the interface, all methods (`Before`/`After`/`Finally`/`Error`) need to be defined.
+To avoid defining empty functions, make use of the `UnimplementedHook` struct (which already implements all the empty functions).
```

Comment:
Is the accurate?

Reply:
Good catch, this was not accurate. Removed.